tests/Unit/ParallelAlgorithms/Events/Test_ObserveTimeStep.cpp

```diff
@@ -227,6 +227,8 @@ void test_observe(const Observer& observer,
   CHECK(std::get<4>(reduction_data.data()) == expected_max_step);
   CHECK(results->reduction_names[5] == "Effective time step");
   CHECK(std::get<5>(reduction_data.data()) == approx(expected_effective_step));
+  CHECK(results->reduction_names[6] == "Minimum Walltime");
+  CHECK(results->reduction_names[7] == "Maximum Walltime");
```

Comment:
Could you add a test in this file that calls the formatter function and checks it against a hard-coded string? This is to make sure the formatting works (number of digits etc).

Reply:
This comment is still open, just to make sure we're not both waiting on each other :)

Reply:
How would this be done since the walltimes are not consistent?

Reply:
You don't need any fancy setup code or realistic numbers. Just call the function explicitly: `Const auto formatted_message = FormatTimeOutput::apply(0.123, ...)` then check the string: `CHECK(formatted_message == "Current sim time is ...")`